### DISCUSSION TITLE
Github updates only on pages that change

### DISCUSSION THREAD

**Discussion by Vaslo** [requarks/wiki#5622] [Help / Questions] [answered]
Love this wiki but just struggle with the Github storage integration.  When I create a new repo for Wiki.js, I can never get it to push to the Github repo unless I make a change in the file.  Here is my process:

I followed the steps listed here to make a key, create a repo on Github, enter the data necessary, etc.:
[Wiki.js Github Integration](https://docs.requarks.io/storage/git)

When I create the blank repo on Github, if I don't add a readme file or just some kind of file, I'll keep getting an error that the branch doesn't exist from Wiki.js if I apply the change and activate the Storage type.  To get around this, I create a blank Readme which initializes the repo on Github and Wiki.js indicates success and that it can connect.  However, whether I use 'bi-directional' or 'push to', nothing happens.  However if I make a change to a page, then just that gets pushed to Github.   Forcing sync doesn't help.

What am I doing wrong?  How do I get an initial push of the existing Wiki.js site and then ensure there are updates for then forward?  Appreciate any help you can provide.

**Comment by NGPixel** [comment, marked as answer]
You need to run the **Add Untracked Changes** action before running the **Force Resync** action, otherwise you're just pushing nothing.
